packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -314,6 +316,17 @@ class CupertinoTextField extends StatefulWidget {
        assert(enableIMEPersonalizedLearning != null),
        keyboardType = keyboardType ?? (maxLines == 1 ? TextInputType.text : TextInputType.multiline),
        enableInteractiveSelection = enableInteractiveSelection ?? (!readOnly || !obscureText),
+       contentCommitMimeTypes = contentCommitMimeTypes ??
+            (onContentCommitted == null
+                ? const <String>[] : const <String>[
+              'image/png',
+              'image/bmp',
+              'image/jpg',
+              'image/tiff',
+              'image/gif',
+              'image/jpeg',
+              'image/webp'
+            ]),
```

Comment:
Nit: Could you define this list as a constant somewhere above and leave a comment about why these formats in particular are included?

Reply:
Defined in `editable_text`.dart